build.bash

```diff
@@ -272,8 +303,8 @@ if [[ $hw_platform == "pi-raspios32" ]] || [[ $hw_platform == "pi-raspios64beta"
 
   # Prerequisites
   echo_process "Checking prerequisites... "
-  REQ_COMMANDS="git curl unzip crc32 dos2unix xz"
-  REQ_PACKAGES="git curl unzip libarchive-zip-perl dos2unix xz-utils"
+  REQ_COMMANDS="git curl wget unzip crc32 dos2unix xz"
+  REQ_PACKAGES="git curl wget unzip libarchive-zip-perl dos2unix xz-utils"
```

Comment:
adapt codestyle (local var, camelCase)
Yeah I know it was me ;-)

Reply:
We actually can't use local vars in this case as it is not inside a function. At some point it would probably be good to make this a real function though.